daemon/daemon.go

```diff
@@ -926,14 +926,13 @@ func (d *Daemon) syncLXCMap() error {
 		if added {
 			log.WithField(logfields.IPAddr, ip).Debugf("Adding local ip to endpoint map")
 		}
+		delete(existingEndpoints, ip.String())
```

Comment:
Sign off on your commit. Elaborate on "It seems we are losing endpoint IPs" by providing evidence that backs up your claim.

Reply:
its a test patch to track down a failure.

Reply:
Thanks for clarifying. I will hold off on reviewing in that case.

Reply:
for what its worth I generally don't sign-off test patches because it stops folks from committing them accidentally.